### PR TITLE
Release integration

### PR DESCRIPTION
**Pull request by elanderholm**
**Requirements**

- [X] I have followed the repository's [pull request submission guidelines](../blob/master/README.md#submitting-pull-requests)
- [?] I have updated my user documentation
- [X] I have updated my developer documentation (my integration README)

**Intended availability**
This is a new integration.  We are a partner.

**Describe the solution you've provided**

Our integration is much like other devops types of integrations you support where the integration work is done at releaseapp.io, not on the LaunchDarkly site.  Our integration allows us to create a new LaunchDarkly Environment whenever we deploy an Ephemeral Environment (like a Heroku Review Environment) for our customers.  This gives them a pristine Release Environment connected to a pristine LaunchDarkly Environment for testing before merge to main and deploying to production.  This means a developer or production manager doesn't run the risk of changing flags in a shared LaunchDarkly environment and changing behavior for other people at their company.  When the environment is removed the integration cleans up after itself by removing the LaunchDarkly Environment it created.

**Provide a request preview**

```bash
$ npm run preview release

> @launchdarkly/integration-framework@1.0.0 preview /Users/erik/Projects/integration-framework
> node preview.js "release"

The "release" integration does not make any outbound requests from LaunchDarkly.
```

**Additional context**

### CODE REVIEW

File: integrations/release/manifest.json
Comment (bwoskow-ld) [inline integrations/release/manifest.json:5]: Minor: there's a trailing space at the end of your description.

File: integrations/release/manifest.json
Comment (bwoskow-ld) [inline integrations/release/manifest.json:17]: Can you provide a horizontal image? Seems like you have one such svg on the top left of https://releaseapp.io/.

File: integrations/release/manifest.json
Comment (bwoskow-ld) [inline integrations/release/manifest.json:4]: Minor: to be consistent with the language on other [integration cards](https://app.launchdarkly.com/default/integrations) let's change this to:

```suggestion
  "overview": "Manage LaunchDarkly environments when deploying Release Environments.",
```

**Review by bwoskow-ld — CHANGES_REQUESTED** [review]
Hi @elanderholm,

Very cool. Thanks for submitting!

In addition to my minor inline comments, I noticed that you have lint errors right now; per [CircleCI](https://app.circleci.com/pipelines/github/launchdarkly/integration-framework/12/workflows/98940536-62d6-4b02-a17f-a16e8d291074/jobs/36) there is whitespace at the end of your `square.svg` file.

Once this change is approved and merged we can enable the Release card on the Integrations page for your particular LaunchDarkly account so that you can see it before anyone else can.

**Comment by bwoskow-ld** [comment]
Hi @elanderholm ,

I wanted to check in and make sure you saw my feedback above.

Additionally, is there a way we could try out your integration?

Cheers,
@bwoskow-ld

**Comment by elanderholm** [comment]
HI Ben-
  No i didn't see the original email.  It got filed under github in gmail
and lost forever, i just happened to be looking at my phone when this one
came in.  Sometimes gmail is kind of the worst..for various reasons.
Anyway, I will take a look at the comments and the proposed changes and get
fixes for those.

You absolutely can take a look at the integration.  I actually got your
docs running in Release after forking the branch, with a small couple of
changes I can walk your through.  I'll send you over a link of it running.
We have a free account and it would be awesome if you guys would try it
out, happy to help you with any challenges.
Erik

**Comment by elanderholm** [comment]
Hi Ben-
  I did all the text changes.  Unfortunately, for the first few I thought I
needed to do them myself and didn't realize I could accept them right in
Github, sorry about that.  But, hopefully I got them all.

  Let me know if you or someone on your team would like to hop on a call
this week and we can show you how we got your docs running in Release, it's
as easy or easier then Netlify.  And, once that's up I can show you the
integration too or I can show you the integration on another app.

Thanks
Erik

**Comment by bwoskow-ld** [comment]
Hmm @elanderholm I don't see any new commits on this pull request -- are you sure that you pushed your latest changes?

**Comment by elanderholm** [comment]
> Hmm @elanderholm I don't see any new commits on this pull request -- are you sure that you pushed your latest changes?

Sorry, I was working on the other pull request for the LaunchDarkly-Docs.  I didn't realize from the email which PR this was referring too.  I'll blame Monday..why not?  Working on this now.

**Comment by elanderholm** [comment]
So, I think i have everything except I don't know what to do about the CircleCi error.  I ran the tests locally with npm test and everything looks good.  I looked at the SVG file, but I'm not really sure how to fix that.  We generated the svg files from sketch so i'm not really sure what to do on that one.

**Comment by bwoskow-ld** [comment]
You can manually edit the two svg files to remove the trailing whitespace at the end of the files. That should fix the lint error.

**Comment by elanderholm** [comment]
> You can manually edit the two svg files to remove the trailing whitespace at the end of the files. That should fix the lint error.

I did try that.  I regenerated them from Sketch, hopefully that worked.  It definitely generated them with different ViewBox sizes this time, hopefully that sorts it.

**Comment by bwoskow-ld** [comment]
I still see trailing whitespace. You can manually edit the svg text in the GitHub editor within this pull request. I don't have permission to do so myself since your source branch is on your remote repository, but you should be able to do so.

**Comment by bwoskow-ld** [comment]
Your lint errors are now resolved and the build passes! I'll go ahead and approve this PR now. We won't enable it in production until your [docs](https://github.com/launchdarkly/LaunchDarkly-Docs/pull/118) are approved and available though, otherwise they'll be a dead link on the integration card.

Regarding your docs PR -- it looks like there are a few unresolved threads, but I can't tell if those have been edited elsewhere. If so, go ahead and resolve the comments, and then re-request review from the approver.

**Review by bwoskow-ld — APPROVED** [review]

**Comment by elanderholm** [comment]
> Your lint errors are now resolved and the build passes! I'll go ahead and approve this PR now. We won't enable it in production until your [docs](https://github.com/launchdarkly/LaunchDarkly-Docs/pull/118) are approved and available though, otherwise they'll be a dead link on the integration card.
> 
> Regarding your docs PR -- it looks like there are a few unresolved threads, but I can't tell if those have been edited elsewhere. If so, go ahead and resolve the comments, and then re-request review from the approver.

Sweet!  Sketch makes those svgs in a way that makes it hard to adjust them correctly.  I never did figure it out, but I found https://petercollingridge.appspot.com/svg-editor/ and it's awesome!  Drop in an SVG and tell it to remove the whitespace and it sorted it out!

The other pull request is where I made some of the changes locally and some in the github UI.  Thanks for your help.

**Pull request merged**
